### PR TITLE
Manases -> AutoMapper added, BookRepository completed, InMemory data added for testing, three endpoints (get all, get by id, and create)

### PR DESCRIPTION
**Pull request by ManasesLovera**
A few changes added to the BookService:

## BookService:

Book repository added
InMemory data for testing instead of using database
3 endpoints added to the book controller (get all, get by id, post/create)

## OrderService

SQL relationships using EntityFrameworkCore

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -15,4 +15,5 @@ USERSERVICES_PORT = 3031
 ORDERCATALOG_PORT = 3032
 ORDERSERVICE_PORT = 3033
 NOTIFICATIONSERVICE_PORT = 3034
-PAYMENTSERVICE_PORT = 3035
\ No newline at end of file
+PAYMENTSERVICE_PORT = 3035
+BOOKSERVICE = 3036
```
Comment (RicardoColladoRothschild) [inline example.env]: Just add the "port" so we know this is a port reference variable (for someone new, if they found this env variable in other part of the code
"BOOKSERVICE_PORT"
It just might looks (rendundante) but its a clarification for future coders in the project.

**Review by RicardoColladoRothschild — COMMENTED** [review]
Just add the "port" so we know this is a port reference variable (for someone new, if they found this env variable in other part of the code
"BOOKSERVICE_PORT"
It just might looks (rendundante) but its a clarification for future coders in the project.

File: example.env (hunk 1 above)
Comment (RicardoColladoRothschild) [reply example.env]: @ManasesLovera

**Comment by ManasesLovera** [comment]
Thanks for the review, fixed, let me know if anything else is needed

**Pull request merged**